src/main/java/com/github/lipinskipawel/board/ai/bruteforce/SmartBoardEvaluator.java

```diff
@@ -0,0 +1,26 @@
+package com.github.lipinskipawel.board.ai.bruteforce;
+
+import com.github.lipinskipawel.board.ai.BoardEvaluator;
+import com.github.lipinskipawel.board.engine.BoardInterface;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+
+public class SmartBoardEvaluator implements BoardEvaluator {
+
+    private static int CONST = 10;
```

Comment:
Is it possible to remove static code and put it into the evaluating method? 
I have a feeling that this code could be optimized really well.
Do you see any usage in the future of this static code?

Reply:
I removed it. If your comment also applies to POINTS I think it's better to initialize values once and just read it instead initializing every time.